mdpow/analysis/solvation.py

```diff
@@ -56,21 +56,38 @@ def __init__(self, solute: EnsembleAtomGroup, solvent: EnsembleAtomGroup, distan
         self._dists = distances
 
     def _prepare_ensemble(self):
-        self._col = ['distance', 'solvent', 'interaction',
-                     'lambda', 'time', 'N_solvent']
```

Comment:
Are you now returning distances? If so, better write a new class and leave the old one as-is.

Reply:
@orbeckst 
I agree, because they are doing different things. Can they both exist as separate classes within the same module?